client/my-sites/stats/stats-subscribers-overview/index.tsx

```diff
@@ -10,14 +10,14 @@ import {
 // array of indices to use to calculate the dates to query for
 const cardIndices = [ 0, 30, 60, 90 ];
 
-interface SubscribersData {
+interface subscribersDataArrays {
```

Comment:
```suggestion
interface SubscribersData {
```

I can see we use capital letters for interfaces in almost all the places, so maybe we want to stick with that 😸